FAQ.md

```diff
@@ -1,37 +1,70 @@
+The ml5.js license requires users to comply with the ml5.js Code of Conduct for the first three years after the commit is made to the repo. After three years, the ml5.js license “decays” into an MIT license.
 
 As long as the ml5.js library is in active development, it is likely that there will always be parts of the library that are licensed under the ml5.js license. As a result, users of the ml5.js library will be required to comply with the Code of Conduct.
 
 However, if the ml5.js library ceases to be actively developed, eventually all of the commits will be more than three years old. At that point the entire library will be licensed under the MIT license.
 
-This achieves two goals. First, it makes sure that users are not tied to an inactive process.  The ml5.js license relies on an active Code of Conduct Committee to enforce the license.  If the library is no longer being actively developed, it is likely that the Code of Conduct Committee is also no longer active as well.  Instead of binding users to comply with a process that no longer exists, decaying to the MIT license allows users to comply with its much more straightforward terms.
+This achieves two goals. First, it makes sure that users are not tied to an inactive process. The ml5.js license relies on an active Code of Conduct Committee to enforce the license. If the library is no longer being actively developed, it is likely that the Code of Conduct Committee is also no longer active as well. Instead of binding users to comply with a process that no longer exists, decaying to the MIT license allows users to comply with its much more straightforward terms.
 
-Second, it makes revising or changing the license easier.  The ml5.js license is an experiment in ethical licensing.  If it turns out we made a horrible mistake, we only have to wait for three years until it disappears.  
+Second, it makes revising or changing the license easier. The ml5.js license is an experiment in ethical licensing. If it turns out we made a horrible mistake, we only have to wait for three years until it disappears.
 
 ## Who is on the Code of Conduct Committee and how is the Committee selected?
 
-The Code of Conduct Committee is responsible for maintaining the Code of Conduct and reviewing accused violations of the Code of Conduct.  The Committee selection process will be finalized as part of the community review of the proposed license.  Once finalized, the will be incorporated into the Code of Conduct and current members will be listed in the document. 
+The Code of Conduct Committee is responsible for maintaining the Code of Conduct as well as deciding if a given use of ml5.js violates the Code of Conduct. It can make those decisions in response to a specific request, or through an investigation it initiates directly.
+
+The Code of Conduct Committee’s members are:
```

Comment:
Suggestion:
* Thinking about maintainability -- we currently list these members in 2 different places -- the FAQ and in the CoC. Maybe we have a `COMMITTEE.md` that we link to?

Reply:
Addressed in: https://github.com/ml5js/Code-of-Conduct/pull/28/commits/662cce7187c3fbf6ee4741e5eafffe44b4050cc4